editor/plugins/navigation_mesh_editor_plugin.cpp

```diff
@@ -0,0 +1,144 @@
+#include "navigation_mesh_editor_plugin.h"
```

Comment:
Could you add the Godot license header? (copy it from another file and change the filename on the second line)

Reply:
Okay, the update should have this now.